cmake/ObsPluginHelpers.cmake

```diff
@@ -104,6 +104,8 @@ macro(find_qt)
     message(FATAL_ERROR "Neither Qt6 nor Qt5 found.")
   endwhile()
 
+  set(QT_VERSION "${_QT_VERSION}")
+
```

Comment:
Why set QT_VERSION again? It should already be set at this point.

Reply:
Yep that's incorrect, `QT_VERSION` should stay as-is (because it's the publicly configurable value). Just checking for `_QT_VERSION` should be fine, because it's either set, or the script throws a fatal error before that.

Reply:
@cg2121 please take another look at this and maybe update for OBS 29, we could merge this then.